src/executor/LanceurExecutor.java

```diff
@@ -32,7 +32,7 @@ public Integer call() throws Exception {
 		try {
 			Integer retour = f.get();
 			System.out.printf("j'ai eu le retour %s%n", retour);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)